apps/native-component-list/src/navigation/RootNavigation.tsx

```diff
@@ -18,8 +18,8 @@ export const linking = {
       main: {
         initialRouteName: 'apis',
         screens: {
-          apis: MainNavigators.apis.linking,
-          components: MainNavigators.components.linking,
+          apis: MainNavigators.apis?.linking,
+          components: MainNavigators.components?.linking,
```

Comment:
Once `undefined` is removed from `MainNavigators` we're ready to revert this change
```suggestion
          apis: MainNavigators.apis.linking,
          components: MainNavigators.components.linking,
```